### DISCUSSION TITLE
Is there a demo dataset for practice?

### DISCUSSION THREAD

**Discussion by reallhw** [muscbridge/PyDKE#300] [Ideas]
The program seems very complicated and runs at a very long time. It would be great if you can provide some demo datasets, so our beginners could learn it easier and more confidently 💯

**Comment by TheJaeger** [comment]
Hello! The large number of calculations performed to derive accurate diffusion and kurtosis measures definitely takes a while. There are plenty of diffusion MR datasets available online, most of which should be compatible with PyDesigner. A good place to start would be here https://github.com/muschellij2/open_neuro.

**Reply by reallhw** [reply]
Bravo!  Thank you for the reply and link!
